public/app/features/alerting/unified/components/MenuItemPauseRule.tsx

```diff
@@ -19,7 +22,10 @@ interface Props {
  * and triggering API call to do so
  */
 const MenuItemPauseRule = ({ rule, onPauseChange }: Props) => {
-  const { group } = rule;
+  // we need to fetch the group again, as maybe the group has been filtered
+  const [getGroup] = alertRuleApi.endpoints.rulerRuleGroup.useLazyQuery(); // shoukd we skip cache here?
```

Comment:
Using a lazy query will always skip the cache unless you tell it not to, so I think this is right :)

Reply:
trueee....at the beginning i was not using lazyquery and i left the comment 😅